zaza/charm_lifecycle/func_test_runner.py

```diff
@@ -232,9 +238,10 @@ def parse_args(args):
     parser.add_argument('--dev', dest='dev',
                         help='Just run dev test(s)',
                         action='store_true')
-    parser.add_argument('-b', '--bundle', dest='bundle',
+    parser.add_argument('-b', '--bundle', dest='bundles',
                         help='Override the bundle to be run',
```

Comment:
Should we extend this description a bit?

Reply:
Incidentally, changing the type does include usage directions that include multiple bundles:

```
usage: func_test_runner.py [-h] [--keep-model] [--smoke] [--dev] [-b BUNDLES [BUNDLES ...]] [-f] [--log LOGLEVEL] [--test-directory TEST_DIRECTORY]

optional arguments:
  -h, --help            show this help message and exit
  --keep-model          Keep model at the end of the run
  --smoke               Just run smoke test(s)
  --dev                 Just run dev test(s)
  -b BUNDLES [BUNDLES ...], --bundle BUNDLES [BUNDLES ...]
                        Override the bundle to be run
  -f, --force           Pass --force to the juju deploy command
  --log LOGLEVEL        Loglevel [DEBUG|INFO|WARN|ERROR|CRITICAL]
  --test-directory TEST_DIRECTORY
                        Set the directory containing the test config (test.yaml etc).
```

I'll update it to say "Override the bundles(s) to be run" as I do think that would be more clear.